crates/viewer/re_space_view_map/src/map_space_view.rs

```diff
@@ -476,39 +476,64 @@ fn handle_ui_interactions(
     }
 }
 
+/// Return http options for tile downloads.
+///
+/// On native targets, it configures a cache directory.
+fn http_options() -> walkers::HttpOptions {
+    if cfg!(target_arch = "wasm32") {
+        Default::default()
+    } else {
+        let cache = directories::ProjectDirs::from("io", "rerun", "Rerun").map(|dirs| {
```

Comment:
Why is different from https://github.com/rerun-io/rerun/blob/2de386bfcb0b0b34b7b97467c3813f7be9b20645/crates/utils/re_analytics/src/native/config.rs#L127 ?

Reply:
I think we should be using https://docs.rs/eframe/latest/eframe/fn.storage_dir.html instead.

Though it would be unfortunate to depend on `eframe` here, perhaps we could pass it by argument (from `ViewerContext`), so that all our on-disk-caches (I'm sure there will be more) use the same storage place.